src/sentry/static/sentry/app/views/onboarding/project/platformiconTile.jsx

```diff
@@ -1,25 +1,104 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import classnames from 'classnames';
+import styled, {css} from 'react-emotion';
 
-class PlatformiconTile extends React.Component {
-  static propTypes = {
-    platform: PropTypes.string,
-    className: PropTypes.string,
-  };
-
-  render() {
-    const {platform, className} = this.props;
-
-    return (
-      <li
-        className={`platform-tile list-unstyled ${platform} ${platform.split(
-          '-'
-        )[0]} ${className}`}
-      >
-        <span className={`platformicon platformicon-${platform}`} />
-      </li>
-    );
+// [platformName => [background, forground]
+const platformColors = [
+  ['python', ['#3060b8']],
+  ['python-django', ['#57be8c']],
+  ['javascript', ['#ecd744', '#111']],
+  ['javascript-react', ['#2d2d2d', '#00d8ff']],
+  ['javascript-ember', ['#ed573e', '#fff']],
+  ['ruby', ['#e03e2f', '#fff']],
+  ['rails', ['#e03e2f', '#fff']],
+  ['javascript-angular', ['#e03e2f', '#fff']],
+  ['java', ['#ec5e44']],
+  ['php', ['#6c5fc7']],
+  ['node', ['#90c541']],
+  ['app-engine', ['#ec5e44']],
+  ['csharp', ['#638cd7']],
+  ['go', ['#fff', '#493e54']],
+  ['elixir', ['#4e3fb4']],
+];
+
+const getColorStyles = props =>
+  !props.monoTone &&
+  css`
+    background: #625471;
+    color: #fff;
+
+    ${platformColors.map(
```

Comment:
hot damn lol. are you sure we can't do this more like `background: platformColors[platform].background`?

Reply:
Ah yeah, idk why I didn't just do that 😬

Reply:
Okay, fixed this way up. Actually doesn't even use any of the classes from platformicons now. Ideally we should convert these platform icons into a sprite set as well, or somehow merge them with the other svg platform icons we have.